internal/integration/api/reboot.go

```diff
@@ -72,7 +72,7 @@ func (suite *RebootSuite) TestRebootNodeByNode() {
 
 			var uptimeAfter float64
 
-			suite.Require().NoError(retry.Constant(3 * time.Minute).Retry(func() error {
+			suite.Require().NoError(retry.Constant(10 * time.Minute).Retry(func() error {
```

Comment:
`ctx` timeout above should be bumped to be be at least 10 minutes to make this reasonable, otherwise `ctx`/`nodeCtx` is going be canceled before retry stops retrying

Reply:
bumped ctx to 10 mins.